easystacks/software.eessi.io/2023.06/espresso-eb-5.1.0-dev.yml

```diff
@@ -3,4 +3,3 @@ easyconfigs:
       options:
         software-commit: 4b9ccc45713137eb16e5f19e5a313b5342080c75
         include-easyblocks: easyconfigs/*/*/*.py
-  - HeFFTe-2.4.1-foss-2023b.eb
```

Comment:
This should have been before ESPResSo, not after

Reply:
Noted. Once heFFTe is available, I'll revert the commit that removed the EESSI heFFTe module and shift this line above.